great-idea/index.html

```diff
@@ -14,10 +15,90 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
+<body>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
       
-    
-  </body>
+        <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </nav>
+    </header>
+
+    <section class="CTA">
+      <div class="IOD-left">
+        <h1>
+          Innovation<br>
+          On<br>
+          Demand<br>
+        </h1>
+        <div class="IOD-button">
+          Get Started
+        </div>
```

Comment:
```suggestion
         <div class="IOD-button">
              Get Started
        </div>
```

you would want to use a <button> here you can use a div but you have to do extra work to style and make it more interactable